LayoutTests/http/tests/workers/reportingObserverNonNull.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="../../resources/js-test-pre.js"></script>
```

Comment:
In modern tests, we include `js-test.js`.

`js-test-pre.js` + `js-test-post.js` is the old style. And including one without the other (like you did) is never correct.